doc/integrating/integration.md

```diff
@@ -136,3 +132,43 @@ const model = new Model(data, {
   },
 });
 ```
+
+## Custom notifications
+
+You can define your own notification callbacks by updating the Notification store (see [`stores`](/src/store_engine/README.md)).
+
+Here is a simple example with a minimal setup.
+
+```typescript
+const { Spreadsheet, Model, stores } = o_spreadsheet;
+const { Component, whenReady, xml } = owl;
+
+const { useStoreProvider, useStore, NotificationStore } = stores;
+
+class Wrapper extends Component {
+  static template = xml/* xml */ `<div><Spreadsheet model="model"/></div>`;
+  static components = { Spreadsheet };
+
+  model = new Model();
+
+  setup() {
+    useStoreProvider();
+    const notificationStore = useStore(NotificationStore);
+    notificationStore.update({
```

Comment:
this would be circling back to where we were 3 years ago. I don't really mind (agree with the ease of use) but I have 2 points:
- The store approach allows to have at least one concrete example on how to interract with stores for external users that want to set an integration
- Back in the days, we wanted to alleviate the number of props and it feels way better to only have to provide model (look at version 15.0 ... 
![image](https://github.com/odoo/o-spreadsheet/assets/10333790/4bc4ddd7-8a7e-4dbe-b15b-81b2f85ee72d)
)